code/modules/clothing/head/helmet.dm

```diff
@@ -24,6 +24,9 @@
 	max_heat_protection_temperature = HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.7
 	w_class = ITEM_SIZE_NORMAL
+	matter = MAT_STEEL
+	matter = list(MAT_GLASS = MATTER_AMOUNT_REINFORCEMENT)
```

Comment:
Was this supposed to be in the helmet/space entry?

Reply:
Nope